specifications/definition/conditional/boolean/definition.conditional.boolean.schema.json

```diff
@@ -11,7 +11,7 @@
       "type": "boolean"
     },
     {
-      "$ref": "http://gov.uk/schema/v1.0.0/condition"
+      "$ref": "http://gov.uk/schema/v1.0.0/definition/condition/definition.condition.schema.json"
```

Comment:
Should be `"$ref": "http://gov.uk/schema/v1.0.0/condition/condition.schema.json"` but reintroduces complex circularity (affecting many of the schemas). Try `any` `all` `exactly`